Runtime/Scripts/Internal/FFIClient.cs

```diff
@@ -108,23 +109,46 @@ static void Dispose()
             Utils.Debug("FFIServer - Disposed");
         }
 
-        internal static FfiResponse SendRequest(FfiRequest request )
+        async internal static Task<FfiResponse> SendRequest(FfiRequest request )
         {
-            
+
             var data = request.ToByteArray(); // TODO(theomonnom): Avoid more allocations
-            
-            
-            
-            FfiResponse response = null;
-            unsafe
+
+
+            return await Task.Run(() =>
```

Comment:
it might be redundant to schedule one more task (on the thread pool) as this method can be already called from the background thread.
Please check all possible invocations.